Chart.yaml

```diff
@@ -3,7 +3,7 @@
 
 apiVersion: v2
 name: vault
-version: 0.25.0
+version: 0.25.1
```

Comment:
Please can you revert this? We make the call on what the next version will be at release time, so no need to edit it in the meantime. Thanks!